src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
```

Comment:
Mind code formatting: first line has 4 space indentation, while following only 2 (2 is correct) - please be consistent

Reply:
I didn't fully understand what you mean, everything seems to be fine with the formatting. Otherwise, I wouldn't be allowed to start the lint if there were any errors in the indents